backend/apps/incidences/views.py

```
@@ -149,20 +149,26 @@ def notifications(self, request):
-            
-            # Si el estudiante reporta en habitación y no puso número, 
-            # le ponemos el de su perfil (o el "3º A" por ahora)
-            if location_type == 'habitacion' and not room_number:
-                room_number = "3º A"
-            
-            serializer.save(student=user, room_number=room_number)
-        
+            if membership and membership.bedroom:
+                assigned_room = membership.bedroom
+            else:
+                logger.warning(f"Student {user.id} tried to report a room issue but has no assigned room.")
```

Ruff (0.15.6)

[error] Undefined name `logger`

(F821)